cvpy/tests/test_image.py

```diff
@@ -430,7 +434,7 @@ def test_mask_encoded_image_encoded_mask(self):
         new_img = self.s.CASTable('new_img', replace=True)
 
         # Construct Image object
-        image = Image(cas_session=self.s)
+        image = NaturalImageTable(cas_session=self.s)
```

Comment:
Hi @mumuMia , did you run the tests? I am not sure how did this work, if there is no cas_session parameter in NaturalImageTable class constructor?